lib/base-config.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
 const { isCI } = require('ci-info');
+const { defaults } = require('jest-config');
 
 module.exports = () => ({
+    ...defaults,
     transform: {
         '\\.js$': require.resolve('babel-jest'),
```

Comment:
Should we be spreading the `...defaults.transform` here as well as in other properties? This is something to reflect on: on one hand, we will be applying the defaults to all these properties if Jest changes them. On the other hand, it may cause unexpected issues as the new Jest defaults might no be what we want.